tests/tck/features/optimizer/IndexScanRule.feature

```diff
@@ -20,25 +20,25 @@ Feature: Match index selection
-      | 13 | Filter      | 7            |                                                                                                                        |
-      | 7  | Project     | 6            |                                                                                                                        |
-      | 6  | Project     | 5            |                                                                                                                        |
-      | 5  | Filter      | 15           |                                                                                                                        |
-      | 15 | GetVertices | 11           |                                                                                                                        |
-      | 11 | IndexScan   | 0            | {"indexCtx": {"columnHints":{"scanType":"RANGE","column":"name","beginValue":"\"Tim Duncan","endValue":"\"Yao Ming"}}} |
-      | 0  | Start       |              |                                                                                                                        |
+      | id | name        | dependencies | operator info                                                                                                              |
+      | 10 | Project     | 13           |                                                                                                                            |
+      | 13 | Filter      | 7            |                                                                                                                            |
+      | 7  | Project     | 6            |                                                                                                                            |
+      | 6  | Project     | 5            |                                                                                                                            |
+      | 5  | Filter      | 15           |                                                                                                                            |
+      | 15 | GetVertices | 11           |                                                                                                                            |
+      | 11 | IndexScan   | 0            | {"indexCtx": {"columnHints":{"scanType":"RANGE","column":"name","beginValue":"\"Tim Duncan\"","endValue":"\"Yao Ming\""}}} |
+      | 0  | Start       |              |                                                                                                                            |
 
   Scenario: or filter embeding
     When profiling query:
       """
       MATCH (v:player)
       WHERE
-        v.name<="Aron Baynes"
-        or v.name>"Yao Ming"
-        or v.name=="Kobe Bryant"
-        or v.age>40
+      v.name<="Aron Baynes"
+      or v.name>"Yao Ming"
+      or v.name=="Kobe Bryant"
+      or v.age>40
```

Comment:
Is this a better format?

Reply:
This is automatically formatted by vscode. It was not in this format before, but lint would report an error, so I formatted it with vscode.